common/lib/capa/capa/responsetypes.py

```diff
@@ -371,7 +371,7 @@ def make_hint_div(self, hint_node, correct, student_answer, question_tag,
             hints_wrap = u'<div class="{0}">{1}</div>'.format(QUESTION_HINT_MULTILINE, hints_wrap)
         label_wrap = ''
         if label:
-            label_wrap = u'<div class="{0}">{1}: </div>'.format(QUESTION_HINT_LABEL_STYLE, label)
+            label_wrap = u'<span class="{0}">{1}: </span>'.format(QUESTION_HINT_LABEL_STYLE, label)
```

Comment:
These python format templates should use names rather than numbers

Example here: https://github.com/edx/edx-platform/pull/13018/files#diff-33bfdd5ef1f14aeda29ebad5a6a35bc5R122

Also here: https://pyformat.info/

Reference from when Christina pointed this out in my PR: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html#use-named-placeholders